src/http/formdata/builder.cr

```diff
@@ -37,15 +37,15 @@ module HTTP::FormData
 
     # Adds a form part with the given *name* and *value*. *Headers* can
     # optionally be provided for the form part.
-    def field(name, value, headers : HTTP::Headers = HTTP::Headers.new)
-      file(name, IO::Memory.new(value), headers: headers)
+    def field(name : String, value, headers : HTTP::Headers = HTTP::Headers.new)
+      file(name, IO::Memory.new(value.to_s), headers: headers)
     end
 
     # Adds a form part called *name*, with data from *io* as the value.
     # *Metadata* can be provided to add extra metadata about the file to the
     # Content-Disposition header for the form part. Other headers can be added
     # using *headers*.
-    def file(name, io, metadata : FileMetadata = FileMetadata.new, headers : HTTP::Headers = HTTP::Headers.new)
+    def file(name : String, io, metadata : FileMetadata = FileMetadata.new, headers : HTTP::Headers = HTTP::Headers.new)
```

Comment:
why not also restrict `io : IO`?

Reply:
I don't know :(

Reply:
I mean, if not putting a type restriction led to a bug, we should always put type restrictions when possible.